tests/integration/cgroups_fetcher_mock_test.go

```diff
@@ -18,7 +18,7 @@ type CgroupsFetcherMock struct {
 
 // NewCgroupsFetcherMock creates a new cgroups data fetcher.
 func NewCgroupsFetcherMock(hostRoot string, time time.Time, systemUsage uint64) (*CgroupsFetcherMock, error) {
-	cgroupsFetcher, err := raw.NewCgroupsFetcher(hostRoot)
+	cgroupsFetcher, err := raw.NewCgroupsV1Fetcher(hostRoot)
```

Comment:
Do you think it woud make sense this mock to support `CgroupsV2Fetcher` as well? 🤔 Anyway, this can be addressed when implementing the tests for `cgroupsV2Fetcher`.

Reply:
As you said, we can find out the best approach once we implement `cgroupsV2Fetcher` tests. At the moment, I chose to use the `v1` builder instead of the generic one because only the `v1` is being tested.